zulipterminal/ui_tools/views.py

```diff
@@ -1023,7 +1026,28 @@ def __init__(self, controller: Any, msg: Message, title: str) -> None:
                     grouped_reactions[reaction] = user
             msg_info.append(('Reactions', list(grouped_reactions.items())))
 
+        # Render the category using the existing table methods if links exist.
+        if message_links:
+            msg_info.append(('Message Links', []))
```

Comment:
I am currently showing `message_links` at the bottom since it is straightforward to implement and we were planning to move reactions away from `MsgInfoView` in #707. Though, I am not sure when that would be integrated. Thoughts?